### PR TITLE
Implement role support for manual multiple user invite

### PR DESCRIPTION
**Pull request by dasuni-30**
Screenshot 2023-11-06 at 23 14 13

### Related Issues
- https://github.com/wso2/product-is/issues/17284

### Related PRs
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx [hunk 1]
```diff
@@ -259,12 +283,17 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
...
-        setConfigureMode(MultipleInviteMode.MANUAL);
-    }, [ ]);
+    // Validate the input string is an email address.
+    const validateEmail = (emailList: string[]) => {
+
+        const emailValidation: boolean = FormValidation.email(emailList[emailList.length-1]);
+
+        if (!emailValidation) {
+            setIsEmailDataError(true);
+            setEmailDataError("Enter a valid email address");
```
Comment (Rashmini) [inline apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx:293]: Shall we localize this string?

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx [hunk 2]
```diff
@@ -1105,9 +1136,37 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
...
 
+            const user: { value: string; } = {
+                value: `bulkId:bulkId:${email}:${asyncOperationID}`
+            };
+            
+            users.push(user);
             operations.push(SCIMBulkOperation);
         });
 
+        // Create the roles record
```
Comment (Rashmini) [inline apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx:1147]:
```suggestion
        // Create the roles record.
```

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx [hunk 3]
```diff
@@ -235,6 +252,13 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
         getUserStoreList();
     }, [ userstore ]);
 
+    /**
+     * Set the default multiple invites option.
+     */
+    useEffect(() => {
+        setConfigureMode(MultipleInviteMode.MANUAL);
```
Comment (Achintha444) [inline apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx:259]: Without using this in an `useEffect` shall we directly assign as the initial value ?

```
    const [ configureMode, setConfigureMode ] = useState<string>(MultipleInviteMode.MANUAL);
```

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx (hunk 1 above)
Comment (dasuni-30) [reply apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx]: Fixed with 9fd857accd3f35459c4b56b50a2d17744b271f36.

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx (hunk 3 above)
Comment (dasuni-30) [reply apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx]: Improved with 9fd857accd3f35459c4b56b50a2d17744b271f36.

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx (hunk 2 above)
Comment (dasuni-30) [reply apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx]: Improved with 9fd857accd3f35459c4b56b50a2d17744b271f36.

**Review by Rashmini — APPROVED** [review]

**Review by Achintha444 — APPROVED** [review]

**Pull request merged**